README.md

```diff
@@ -0,0 +1,73 @@
+# openpaas-front
+
+Repository that contains recipes to build & publish OpenPaaS frontends :
+* Inbox
+* Mailto
+* Account
+* Contacts
+* Calendar
+* Calendar Public
+* Admin
+
+Every frontends will be builded separately and available in a single Docker image and served by Nginx. 
+
+Nginx locations can be find in `nginx.conf` file
+
+## Build
+
+Build all frontends on the main branch
+
+```sh
+docker build -t openpaas-front .
+```
+
+### Custom Build
+
+You can override standard build using docker build arguments
+
+#### Available build arguments
+
+---------------------------------------
+
+| Build Arg      | Description |
+| ----------- | ----------- |
+| NODE_VERSION   | NodeJS version        |
+| NGINX_VERSION   | Nginx version        |
+| INBOX_GIT_BRANCH   | Git branch name for inbox frontend        |
```

Comment:
Does your magic github link https://github.com/OpenPaaS-Suite/X/archive/Y also work for git tags ?

Because we will need to build branches, and also to build specific release tags, like "1.3.2" . In that case, the commonly used word to say "any label that can refer to somewhere in a git tree" is called TREEISH ( https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddeftree-ishatree-ishalsotreeish ).

If your current script does only support branches, let's go like that and we'll improve later. If it already support treeish values, please rename X_GIT_BRANCH to X_GIT_TREEISH.